src/sentry/snuba/events.py

```diff
@@ -0,0 +1,123 @@
+from __future__ import absolute_import
+
+from enum import Enum
+from collections import namedtuple
+
+Column = namedtuple("Column", "event_name transaction_name discover_name alias")
```

Comment:
Long term we can likely remove the `transaction_name` as I don't think we have any present/known use cases for accessing the transactions dataset directly.

Reply:
Great